manifests/d/DrStefanFreischlad/Filius/2.0.1/DrStefanFreischlad.Filius.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: DrStefanFreischlad.Filius
+PackageVersion: 2.0.1
+PackageLocale: en-US
+Publisher: Dr. Stefan Freischlad

```

```suggestion
Publisher: Stefan Freischlad
```
This must match Control Panel.